precalculus/exercises/outcomes/FN/FN1/template.xml

```diff
@@ -9,17 +9,89 @@
     
     
     <knowl>
-        <content>
+        <intro>
             <p>
-                Task 
+                Explain why each of the following is or is not a function.
             </p>
+        </intro>
+                <!-- {{#tasks}} -->
+                <!-- {{#map_task}} -->
+                <knowl>
+                    <content>
+                    <p>
+                    <image
+                        source="assets/FN1/generated/{{__seed__}}/mapping.png"
+                        description="Solution to inequality."/>
+                    </p>
+                    </content>
+                    <outtro>
+                        <!-- {{#map_function}}-->
+                        <p>This mapping is a function.</p>
+                        <!-- {{/map_function}}-->
+                        <!-- {{^map_function}}-->
+                        <p>This mapping is not a function.</p>
+                        <!-- {{/map_function}}-->
+                    </outtro>
+                </knowl>
+                <!-- {{/map_task}} -->
+                <!-- {{#table_task}} -->
+                <knowl>
+                    <content>
+                    <p>Table goes here.  </p>
+                    <table>
+                    <tabular>
+                        <row header="yes" bottom="minor">
+                            <cell><m>x</m></cell> 
+                            <cell><m>y</m></cell>
+                        </row>
+                        {{table_rows}}
+                    </tabular>
+                </table>
```

Comment:
This is the template code for the troublesome table.  I based this off of the code for [Activity 2.1.9(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/FN1.html#FN1-determine-if-a-function-3)

Reply:
table/tabular are not supported by SpaTeXt. Instead I think the best solution is to create something in an `<me/>` using LaTeX

Reply:
```xml
            <me>
\begin{array}{c|ccccccc}
x {{#nearbys}}&amp; {{x}}{{/nearbys}} \\\hline
f(x) {{#nearbys}}&amp; \hspace{3em}{{/nearbys}} \\
\end{array}
            </me>
```

https://teambasedinquirylearning.github.io/calculus/2024e/exercises/#/bank/LT2/1/

Reply:
```suggestion
<p>
  <me>
\begin{array}{c|c} 
x &amp; y \\\hline
{{#table_rows}}
{{x}} &amp; {{y}} \\
{{/table_rows}}
\end{array}
  </me>
</p>
```